app/ui/browser/reducers/main-reducers.js

```diff
@@ -37,15 +37,26 @@ function getPageIndexById(state, id) {
 export default function basic(state = initialState, action) {
   switch (action.type) {
     case types.CREATE_TAB:
```

Comment:
What's the use case for `types.CREATE_TAB` vs `types.IPC_COMMAND_CREATE_TAB`? 

It looks like the action creator that uses `types.CREATE_TAB` is only used in `external.js`. It's used in various context menus to create a new tab. But the `BrowserMenu` uses IPC to do the same thing. Even within the context menus in `external.js`, some menu items use `ipcRenderer.send`instead of dispatching actions.

I'm not close enough to the details to know for sure, but it feels like we might want to consolidate those.